pyproject.toml

```diff
@@ -3,22 +3,20 @@ name = "PMP_IV"
 version = "0.0.5"
 description = "Gestor dependencias para objetivo 3 en adelante"
 authors = ["Pablo Morenilla <MauronMP>"]
-readme = ["docs/Gestor_Dependencias.md", "README.md"]
 repository = "https://github.com/MauronMP/PMP_IV/"
-documentation = "https://github.com/MauronMP/PMP_IV/tree/Objetivo-3/docs" #Temporal, después será la rama main
-
-packages = [
-    { include = "pmp_iv/**/*.py" },
-    { include = "pmp_iv/*.py" },
-]
-
+documentation = "https://github.com/MauronMP/PMP_IV/tree/Objetivo-5/docs" 
 
 [tool.poetry.dependencies]
 python = ">=3.8" 
 pyhamcrest = "^2.0.4"
 pytest = "^7.2.0"
+invoke = "^1.7.3"
 
 [tool.poetry.dev-dependencies]
 pytest = "^7.2.0"
+invoke = "^1.7.3"
 
-
+[tool.pytest.ini_options]
+filterwarnings = [
```

Comment:
Debéis de acostumbraros a hacer sólo cosas que estén justifcadas con un issue. Si esto es respuesta a un error (o warning) que habéis observado, la forma correcta de hacerlo es abrir un issue donde explique qué ha pasado y modificar *solo esto* para cerrar el issue.
Tenéis que entender que lo importante de un PR es que lo entienda correctamente la persona que lo revisa. Sin entenderlo, es imposible aprobarlo (o rechazarlo)

Reply:
:arrow_up:

Reply:
En el issue #40 lo explico:
> Una vez creada la imagen de docker #34 y funcionando los tests genera un aviso de que intenta escribir ficheros, pero no tiene permitido, estos son los de caché, pero para ello se ha arreglado en el issue #38, para evitar que aparezca este mensaje en el fichero pyproject.toml:
> 
> ```
> [tool.pytest.ini_options]
> filterwarnings = [
>     "ignore::UserWarning",
> ]
> ```